src/DynamoCore/Configuration/IPreferences.cs

```diff
@@ -75,7 +44,7 @@ public interface IPreferences
         /// <summary>
         /// Indicates whether background preview is active or not.
         /// </summary>
-        [Obsolete("Property will be deprecated in Dynamo 2.0, please use BackgroundPreviews")]
+        [Obsolete("Property will be deprecated in Dynamo 3.0, please use BackgroundPreviews")]
```

Comment:
do you mean to delete these?

Reply:
Still investigating since I found this property is serialized in setting as well. Not sure about the impact.